web/index.html

```diff
@@ -9,7 +9,7 @@
     <meta charset="utf-8" />
     <link rel="icon" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#141415" />
+    <meta name="theme-color" content="#18181B">
```

Comment:
Changed this because it more closely matches the color of the  gradient we have at the top.